core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -124,61 +124,82 @@ export class ContributorAdminStatsTable implements OnInit {
     return num === 1 ? word : word + 's';
   }
 
-  getCountDisplay(
+  getEditTextForAttribute(
```

Comment:
I don't know what "edit text" means and I also don't know what "attribute" means here. Could you make the function name clearer and possibly add a JSDoc to clarify the contract of the function and its arguments?

Re the following:

> Done I think. I struggled with the meaning of the second sentence, but hopefully what I've done is ok :)

the idea is that if an identifier in the code starts with a verb, it's a method/function, otherwise it's a variable. For example, getXYZ() and isABC() are methods/functions, and itemIsABC or XYZ are variables. We try to stick to this convention in general. (Does that help?)

Reply:
Thanks for the explanation, I've moved the formatting into a service separate from the component, and added some js doc, hopefully it makes more sense now.

Reply:
Done